installer/install.iss

```diff
@@ -2058,28 +2085,28 @@ begin
     end;
 
     (*
-     * Create a custom page for using (Tortoise)Plink instead of OpenSSH
-     * if at least one PuTTY session is found in the Registry.
+     * Create a custom page for using (Tortoise)Plink or a self-supplied OpenSSH instead of bundled OpenSSH
+     * if at least one PuTTY session is found in the Registry or an OpenSSH binary is found on the PATH.
      *)
 
-    if RegGetSubkeyNames(HKEY_CURRENT_USER,'Software\SimonTatham\PuTTY\Sessions',PuTTYSessions) and (GetArrayLength(PuTTYSessions)>0) then begin
-        PuTTYPage:=CreatePage(PrevPageID,'Choosing the SSH executable','Which Secure Shell client program would you like Git to use?',TabOrder,Top,Left);
+    if (RegGetSubkeyNames(HKEY_CURRENT_USER,'Software\SimonTatham\PuTTY\Sessions',PuTTYSessions) and (GetArrayLength(PuTTYSessions)>0)) or
+       (FileSearch('ssh.exe', GetEnv('PATH')) <> '') then begin
+        SSHChoicePage:=CreatePage(PrevPageID,'Choosing the SSH executable','Which Secure Shell client program would you like Git to use?',TabOrder,Top,Left);
 
         // 1st choice
-        RdbSSH[GS_OpenSSH]:=CreateRadioButton(PuTTYPage,'Use OpenSSH','This uses ssh.exe that comes with Git. The GIT_SSH and SVN_SSH'+#13+'environment variables will not be modified.',TabOrder,Top,Left);
+        RdbSSH[GS_OpenSSH]:=CreateRadioButton(SSHChoicePage,'Use bundled OpenSSH',
+            'This uses ssh.exe that comes with Git.',
```

Comment:
Please avoid introducing a coding style that is inconsistent with what is already in this file.